webpack.config.js

```diff
@@ -1,23 +1,70 @@
 const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
+const TerserPlugin = require("terser-webpack-plugin");
+const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const path = require("path");
+const webpack = require("webpack");
+
 const APP_PATH = path.resolve(__dirname, "src");
 
+const isDevelopment = process.env.NODE_ENV === "development";
+const isProduction = process.env.NODE_ENV === "production";
+
+const styleLoaders = {
+  test: /\.s?css$/,
+  loader: [
+    "style-loader",
+    isProduction && {
+      loader: MiniCssExtractPlugin.loader,
+    },
+    "css-loader",
+    "sass-loader",
+    {
+      loader: require.resolve("postcss-loader"),
+      options: {
+        plugins: () => [
+          require("postcss-preset-env")({
+            autoprefixer: {
```

Comment:
should we autoprefix in both modes or just when isProduction?

Reply:
Probably just production